src/dev/jest/config.js

```diff
@@ -44,15 +40,14 @@ export default {
   ],
   modulePathIgnorePatterns: [
     '__fixtures__/',
+    '<rootDir>/packages/kbn-ui-framework/target/',
```

Comment:
Maybe we should just do `/target/` here? I felt I had already included that in some PR, but maybe it's one of my active ones 🙈